backend/tests/api/test_group_tasks.py

```
@@ -0,0 +1,361 @@
+            mock_db, user_id=1, skip=0, limit=10
+        )
+
+        # Verify results
+        assert total == 1
+        assert len(result) == 1
+        assert result[0]["id"] == 123
+        assert result[0]["is_group_chat"] is True
+        assert result[0]["title"] == "Group Chat Task"
+
+    def test_excludes_single_chat_tasks(
+        self, task_service, mock_db, mock_single_chat_task
```

Ruff (0.14.10)

Unused method argument: `mock_single_chat_task`

(ARG002)